pygmt/helpers/utils.py

```diff
@@ -242,6 +244,11 @@ def data_kind(
     >>> data_kind(data=None, required=False)
     'arg'
 
+    The "empty" kind:
+
+    >>> data_kind(data=None)
+    'empty'
```

Comment:
Be more explicit by adding `required=True` here?

```suggestion
    >>> data_kind(data=None, required=True)
    'empty'
```